### ISSUE TITLE
Building with 2.0.2 fails due to missing ssh.app

### ISSUE THREAD

**Issue by mikebveil** [hexpm/hex#976]
Our build pipeline always grabs the latest version of hex. This morning, after the 2.0.2 release, builds starting failing for us with the error `{:ssh, {'no such file or directory', 'ssh.app'}}`. Explicitly grabbing the 2.0.1 release (`mix local.hex 2.0.1 --force`) fixes the problem.

``` > [13/15] RUN set -x;     export MIX_ENV=prod;     cd src/collection_server &&     mix local.hex --force &&     mix local.rebar --force &&     mix clean &&     mix deps.clean --all &&     mix deps.get &&     make install-protobuf &&     mix local.rebar:              
#17 0.286 + export MIX_ENV=prod                                                                                                       
#17 0.286 + cd src/collection_server                                                                                                  
#17 0.286 + mix local.hex --force                                                                                                     
#17 1.575 * creating /collection_server/.mix/archives/hex-2.0.2
#17 1.596 + mix local.rebar --force
#17 2.887 * creating /collection_server/.mix/elixir/1-14/rebar3
#17 2.904 + mix clean
#17 3.698 Could not start Hex. Try fetching a new version with "mix local.hex" or uninstalling it with "mix archive.uninstall hex.ez"
#17 3.710 ** (MatchError) no match of right hand side value: {:error, {:ssh, {'no such file or directory', 'ssh.app'}}}
#17 3.710     (hex 2.0.2) lib/hex.ex:5: Hex.start/0
#17 3.710     (mix 1.14.0) lib/mix/hex.ex:59: Mix.Hex.start/0
#17 3.710     (mix 1.14.0) lib/mix/dep/loader.ex:188: Mix.Dep.Loader.with_scm_and_app/4
#17 3.710     (mix 1.14.0) lib/mix/dep/loader.ex:141: Mix.Dep.Loader.to_dep/3
#17 3.710     (elixir 1.14.0) lib/enum.ex:1658: Enum."-map/2-lists^map/1-0-"/2
#17 3.710     (elixir 1.14.0) lib/enum.ex:1658: Enum."-map/2-lists^map/1-0-"/2
#17 3.710     (mix 1.14.0) lib/mix/dep/loader.ex:358: Mix.Dep.Loader.mix_children/2
#17 3.710     (mix 1.14.0) lib/mix/dep/loader.ex:18: Mix.Dep.Loader.children/0```

**Comment by gianluca-nitti** [comment]
Same exact problem. Will the fix introduced in #977 be released soon?

**Comment by ericmj** [comment]
We plan to have a new release out today.
